ziti/ziti.go

```diff
@@ -710,10 +710,29 @@ func (context *ContextImpl) RefreshService(serviceName string) (*rest_model.Serv
 	return serviceDetail, nil
 }
 
-func (context *ContextImpl) runSessionRefresh() {
+func (context *ContextImpl) runRefreshes() {
 	log := pfxlog.Logger()
-	svcUpdateTick := time.NewTicker(context.options.RefreshInterval)
-	defer svcUpdateTick.Stop()
+	svcRefreshInterval := context.options.RefreshInterval
+
+	if svcRefreshInterval == 0 {
+		svcRefreshInterval = DefaultServiceRefreshInterval
+	}
+	if svcRefreshInterval < time.Second {
+		svcRefreshInterval = time.Second
```

Comment:
Maybe this is too aggressive for a default (when the caller probably made a mistake)? Also a `minRefreshInterval` constant to cover session and service refreshes might make sense.

Reply:
I was thinking that a developer might want to set it this low. So if the dev forgot to set it at all, it will get set to the default value, but if they have set it to something low, we'll cap it at 1 second.

I'll add a constant for minRefreshInterval

Reply:
constant added